r3bdata/ttt10Data/R3BTTTXMappedData.cxx

```diff
@@ -0,0 +1,44 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// -------------------------------------------------------------------------
+// -----                      R3BTTTXMappedData source file            -----
+// -------------------------------------------------------------------------
+
+#include "R3BTTTXMappedData.h"
+
+// -----   Default constructor   -------------------------------------------
+R3BTTTXMappedData::R3BTTTXMappedData()
+    : fDetID(0)
+    , fChannelID(0)
```

Comment:
fChannelID should be fStripID because this is for a silicon detector with 32 strips, right?